### PR TITLE
Handle overrun restart cursor

### PR DESCRIPTION
**Pull request by awmackowiak**

### CODE REVIEW

File: vsltranscore.py [hunk 1]
```diff
@@ -891,6 +891,9 @@ def execute(self):
             ret = self.vap.Dispatch(self.vapCallBack)
             if 0 == ret:
                 time.sleep(0.1)
+            if ret < -3:
+                print(self.vap.error)
+                exit(ret)
```
Comment (ziollek) [inline vsltranscore.py:896]: negative numbers are not proper exit codes in *nix systems

Comment (awmackowiak) [reply vsltranscore.py:896]: Right.

**Comment by awmackowiak** [comment]
@xcir Can you check this PR :) I know that the -3 wouldn't occur but ... It was. So I add handler of this edge case.

**Comment by xcir** [comment]
Thanks for the PR.
I will review the whole thing when I have time, as some of the code is a bit outdated.

**Pull request merged**